js/views/modals/purchase/Purchase.js

```diff
@@ -131,6 +142,7 @@ export default class extends BaseModal {
     return {
       'click .js-goToListing': 'close',
```

Comment:
This is minor, but IMO `goToListing` is the wrong semantics here. We happen to open the Purchase modal above the listing, so closing it will take it to the listing, but it doesn't have to be that way. In theory, the Purchase modal can be opened above anything, so semantically `js-close` (or something to indicate it's just a close) would be more appropriate.